### ISSUE TITLE
Missing items: Necks from Love is in the Air event

### ISSUE THREAD

**Issue by jdelvare** [wowsims/wotlk#2647]
Links to [WOTLK Wowhead](https://wowhead.com/wotlk/items "https://wowhead.com/wotlk/items") for each item:
[Sweet Perfume Broach](https://www.wowhead.com/wotlk/item=51807/sweet-perfume-broach)
And presumably also the other necks from the Love is in the Air event:
[Shard of Pirouetting Happiness](https://www.wowhead.com/wotlk/item=51806/shard-of-pirouetting-happiness)
[Choker of the Pure Heart](https://www.wowhead.com/wotlk/item=51808/choker-of-the-pure-heart)
[Winking Eye of Love](https://www.wowhead.com/wotlk/item=51804/winking-eye-of-love)
[Heartbreak Charm](https://www.wowhead.com/wotlk/item=51805/heartbreak-charm)

**Comment by Saltyc0der** [comment]
They're in the sim. They're just listed under phase 5

**Comment by jdelvare** [comment]
Good catch, I missed that. Well I could have guessed, because they are also (mistakenly) listed as Phase 5 on wowhead (which I suppose is where the simulator fetches its data from, which in turn explains why the mistake propagated there).
So the issue is not missing items but wrong phase information for these items.

**Comment by jarveson** [comment]
These are under the correct dropdown, but tooltip will incorrectly show 'phase 5' till wowhead updates
